desktop/src/main/nl/tudelft/ti2806/riverrush/graphics/entity/RockGraphic.java

```diff
@@ -79,6 +85,9 @@ public void draw(final Batch batch, final float parentAlpha) {
         TextureRegion region = new TextureRegion(tex, 0, 0, TEXTURE_SIZE_X, TEXTURE_SIZE_Y);
         batch.enableBlending();
 
+        Color color = this.getColor();
+        batch.setColor(color.r, color.g, color.b, color.a * parentAlpha);
+
```

Comment:
I assume the fade is not working yet? A possible solution is resetting the color after the batch.draw call (see some of the other graphics classes for example)